front/src/config/i18n/fr.json

```diff
@@ -348,6 +349,18 @@
           "cancelLabel": "Annuler",
           "successLabel": "Effectué"
         }
+      },
+      "setup": {
+        "title": "Scanner de présence",
+        "noConfigLabel": "La configuration n'est pas chargée, merci de réessayer.",
+        "errorLabel": "Une erreur est survenue lors du chargement de la configuration, merci de consulter les logs du navigateur.",
```

Comment:
Do we really want to advertise looking at the log? I've never seen this in any product